app/(saas)/create-project/ProjectCreation/views/GithubOrganizations/components/github-workflow-tutorial-side-panel/components/Help.tsx

```
@@ -0,0 +1,31 @@
+  const { openIntercom } = useIntercom();
+
+  const contactUs = () => {
+    onClose();
+    openIntercom();
+  };
+
+  return (
+    <Card className="flex w-full flex-col items-start justify-start gap-6 p-5">
+      <div className="flex w-full flex-col items-start justify-start gap-2">
+        <TypographyP className="uppercase">
+          What if you can't grant permissions to all organisations?
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)